### DISCUSSION TITLE
Tell me your favorite (fictional) character from your favorite game, anime, show or anything!

### DISCUSSION THREAD

**Discussion by Kotjik** [NAIST-SE/AttractingContributors#415] [General]
I like to know which character left an impression on you. So please add that person/character on my list! The character can be from an anime, a show, a movie, a book, a manga or whatever you like. It can also be a real person, like Julius Cäsar.

Contribute here:
https://github.com/Kotjik/AttractingContributors-FavoriteCharacter

Thank your for your help!

**Comment by aru1702** [comment]
Despite of lately character I have affection to, I prefer to list my childhood character.
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<
